R/unnest_ngrams.R

```diff
@@ -0,0 +1,52 @@
+#' Wrapper around unnest_tokens & ngram
+#'
+#' This function is a wrapper around `unnest_tokens( token = "ngrams" )`.
+#'
+#' @seealso
+#' + [unnest_tokens()]
+#'
+#' @inheritParams unnest_tokens
+#' @inheritParams tokenizers::tokenize_ngrams
+#'
+#' @export
+#' @importFrom dplyr enquo
+#'
+#' @examples
+#' library(dplyr)
+#' library(janeaustenr)
+#'
+#' d <- tibble(txt = prideprejudice)
+#' d
+#' d %>%
+#'   unnest_ngrams(word, txt)
+#'
+unnest_ngrams <- function(
+  tbl,
+  output,
+  input,
+  n = 3L,
+  n_min = n,
+  stopwords = character(),
+  ngram_delim = " ",
+  simplify = FALSE,
+  format = c("text", "man", "latex", "html", "xml"),
+  to_lower = TRUE,
+  drop = TRUE,
+  collapse = NULL
+){
+  format <- match.arg(format)
+  unnest_tokens(tbl,
+                !! enquo(output),
+                !! enquo(input),
+                format = format,
+                to_lower = to_lower,
+                drop = to_lower,
```

Comment:
What's going on here with `drop = to_lower` and `collapse = to_lower`? Am I misunderstanding something, or is that just a typo?

Reply:
that's a typo, sorry. 

Corrected with the last commit